src/main/java/uk/gov/hmcts/reform/idam/client/models/ExchangeCodeRequest.java

```diff
@@ -1,39 +1,14 @@
 package uk.gov.hmcts.reform.idam.client.models;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
 public class ExchangeCodeRequest {
     private String code;
     private String grant_type;
     private String redirect_uri;
     private String client_id;
     private String client_secret;
-
-    public ExchangeCodeRequest(
```

Comment:
I suspect this was here because of the field names being snake cases (that should probably be fixed and json property used instead)
this is fine if you can confirm the generated getters are java style (getGrantType)